_opensearch/snapshot-restore.md

```diff
@@ -156,8 +156,23 @@ Setting | Description
    s3.client.default.proxy.port: 8080 # port for your proxy server
    s3.client.default.read_timeout: 50s # the S3 connection timeout
    s3.client.default.use_throttle_retries: true # whether the client should wait a progressively longer amount of time (exponential backoff) between each successive retry
+   s3.client.default.region: us-east-2 # AWS region to use
    ```
 
+1. (Optional) Alternatively to AWS access and secret keys, you could configure plugin to use AWS IAM roles for service accounts:
+
+   ```bash
+   sudo ./bin/opensearch-keystore add s3.client.default.role_arn
+   sudo ./bin/opensearch-keystore add s3.client.default.role_session_name
+   ```
+
+   Optionally, if AWS access and secret keys are not configured, add to `opensearch.yml` setting (please make sure the file is accessible by `repository-s3` plugin):
```

Comment:
We can remove "Optionally" since we already indicated the step itself is optional. Maybe switch to:

```suggestion
Alternatively, you can add your S3 token to your `opensearch.yml` settings. Make sure your token file is accessible by `repository-s3` plugin:
```

Reply:
This one seems to change the meaning of the sentence: essentially, either AWS access and secret keys or token file is needed, but not both.

Reply:
Understood. Here's an alternative:

```suggestion
 If you don't want to configure AWS access and secret keys, modify the following `opensearch.yml` setting. Make sure the file is accessible by the `repository-s3` plugin:
```